src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier supplier = new FigureSupplier();
+
+        Figure[] figures = {supplier.getRandomFigure(),
+                supplier.getRandomFigure(),
+                supplier.getRandomFigure(),
+                supplier.getDefaultFigure(),
+                supplier.getDefaultFigure(),
+                supplier.getDefaultFigure()
```

Comment:
You should create several random Figures using a loop, and it's better to avoid creating a fixed-size array with hardcoded calls to 'getRandomFigure' and 'getDefaultFigure'. Consider using a loop to populate the array dynamically.